pkg/cmd/pulumi/operations/preview.go

```diff
@@ -593,6 +605,9 @@ func NewPreviewCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVar(
 		&targetDependents, "target-dependents", false,
 		"Allows updating of dependent targets discovered but not specified in --target list")
+	cmd.PersistentFlags().BoolVar(
+		&excludeDependents, "exclude-dependents", false,
+		"Allows ignoring of dependent targets discovered but not specified in --exclude list")
```

Comment:
This isn't new in this PR, but since I'm noticing it.  This should probably be `Allow ...` instead of `Allows ...` to be consistent in the wording here.